code/modules/cargo/goodies.dm

```diff
@@ -0,0 +1,131 @@
+	desc = "Contains a 20-round magazine for the WT-550 Auto Rifle. Each magazine is designed to facilitate rapid tactical reloads."
+	cost = 750 //one of the few single-pack items that who's price per unit is the exact same as the bulk
+	contains = list(/obj/item/ammo_box/magazine/wt550m9)
+
+/datum/supply_pack/goody/sologamermitts
+	name = "Insulated Gloves Single-Pack"
+	desc = "The backbone of modern society. Barely ever ordered for actual engineering."
+	cost = 800
+	contains = list(/obj/item/clothing/gloves/color/yellow)
+
+/datum/supply_pack/goody/gripper_single
+	name = "Gripper Gloves Single-Pack"
+	desc = "A spare pair of gripper gloves. Perfect for when the security vendor is empty (or when you're not actually a security officer)."
+	cost = 600
+	contains = list(/obj/item/clothing/gloves/tackler)
+
+/datum/supply_pack/goody/firstaidbruises_single
+	name = "Bruise Treatment Kit Single-Pack"
+	desc = "A single brute first-aid kit, perfect for recovering from being crushed in an airlock. Did you know people get crushed in airlocks all the time? Interesting..."
+	cost = 330
+	contains = list(/obj/item/storage/firstaid/brute)
+
+/datum/supply_pack/goody/firstaidburns_single
+	name = "Burn Treatment Kit Single-Pack"
+	desc = "A single burn first-aid kit. The advertisement displays a winking atmospheric technician giving a thumbs up, saying \"Mistakes happen!\""
+	cost = 330
+	contains = list(/obj/item/storage/firstaid/fire)
+
+/datum/supply_pack/goody/firstaid_single
+	name = "First Aid Kit Single-Pack"
+	desc = "A single first-aid kit, fit for healing most types of bodily harm."
+	cost = 250
+	contains = list(/obj/item/storage/firstaid/regular)
+
+/datum/supply_pack/goody/firstaidoxygen_single
+	name = "Oxygen Deprivation Kit Single-Pack"
+	desc = "A single oxygen deprivation first-aid kit, marketed heavily to those with crippling fears of asphyxiation."
+	cost = 330
+	contains = list(/obj/item/storage/firstaid/o2)
+
+/datum/supply_pack/goody/firstaidtoxins_single
+	name = "Toxin Treatment Kit Single-Pack"
+	desc = "A single first aid kit focused on healing damage dealt by heavy toxins."
+	cost = 330
+	contains = list(/obj/item/storage/firstaid/toxin)
+
+/datum/supply_pack/goody/toolbox // mostly just to water down coupon probability
+	name = "Mechanical Toolbox"
+	desc = "A fully stocked mechanical toolbox, for when you're too lazy to just print them out."
+	cost = 300
+	contains = list(/obj/item/storage/toolbox/mechanical)
+
+/datum/supply_pack/goody/valentine
+	name = "Valentine Card"
+	desc = "Make an impression on that special someone! Comes with one valentine card and a free candy heart!"
+	cost = 150
+	contains = list(/obj/item/valentine, /obj/item/reagent_containers/food/snacks/candyheart)
+
+/datum/supply_pack/goody/beeplush
+	name = "Bee Plushie"
+	desc = "The most important thing you could possibly spend your hard-earned money on."
+	cost = 1500
+	contains = list(/obj/item/toy/plush/beeplushie)
+
+/datum/supply_pack/goody/camera
```

Comment:
cameras can be printed in autolathe, also no glubs crate?

Reply:
oh fair then, and there's still the solo gamer mitts pack